snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -11,7 +11,7 @@ init_config:
 #  mibs_folder: /path/to/your/mibs/folder
 
 ## @param ignore_nonincreasing_oid - boolean - optional - default: false
-## 
+## XXXX
```

Comment:
What could be the description of this parameter?

Reply:
It's used by the `pysnmp` lib. For some reasons i cannot access the doc from where i am. @zippolyte would you be able to help?

Reply:
There we go: http://snmplabs.com/pysnmp/faq/oids-not-increasing.html?highlight=ignorenonincreasingoid

Reply:
👍